charts/opentelemetry-demo/templates/services.yaml

```diff
@@ -0,0 +1,28 @@
+{{- define "otel.demo.service" }}
```

Comment:
Lets merge the `otel.demo.service`, `otel.demo.deployment`, and `otel-demo.otelcol.config` templates into 1 template file, similar to `charts/opentelemetry-collector/tempaltes/_config.tpl`.  Maybe something like `_objects.tpl`. I could also see an argument for putting `otel-demo.otelcol.config` in a different tpl file since it isn't a k8s object.

The intended goal being all templates are defined in `_*.tpl` files.

Reply:
Thank you for the guidance, learned!